src/python/pants/option/options.py

```diff
@@ -300,7 +291,6 @@ def verify_configs(self, global_config: Config) -> None:
                     """
                 )
             )
-        global_config.verify(section_to_valid_options)
```

Comment:
This was an oversight - config verification has been done in Rust for a while now, but I hadn't removed the python side call...